### ISSUE TITLE
Error on modify issue

### ISSUE THREAD

**Issue by thiagoalmeidasa** [taigaio/taiga-contrib-slack#33]
I got this error when I try to modify some issue:
The integration plugin works fine when I create an issue, but when I modify some issue I got it:

```python
File "/usr/local/lib/python3.4/site-packages/taiga_contrib_slack/tasks.py", line 333, in create_slackhook
    if not data["icon_url"].startswith("http"):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by thnt** [comment]
same here, when change status of task from commit msg

**Comment by bameda** [comment]
It has been fixed in the new version 2.1.1
